webhook/webhook.py

```diff
@@ -0,0 +1,45 @@
+"""Contain all the logic for handling webhooks in a class."""
+import json
+import logging
+
+
+class WebhookHandler:

```

Comment:
The only thing I would still note is, would we want this class to be named Github-specifically? We technically have Slack webhooks as well (though we're using a library to manage them), and might want more in the future.

This isn't a blocker though - don't bother changing it for this PR unless you really want to, but we should keep it in mind as a potential change in the future.

Reply:
Sure, if I were to change it though maybe call it `GitHubWebhookHandler` in `webhook/github.py`?

Reply:
Yeah, sounds good to me.